### ISSUE TITLE
Create ansible role

### ISSUE THREAD

**Issue by kamikaze** [oglueck/to-gentoo#2]
It would be great to have an ansible role for this magic. Also to make an optional backup. Because some systems may not have much space to keep all stuff at once.

**Comment by oglueck** [comment]
Yes why not. One of the challenges is that to-gentoo requires single-user mode. Not sure if you can put a host into single-user reliably with Ansible without losing the network.
But then we could also drop the single-user mode requirement (with a command line switch). It is only there to prevent problems like NFS server locking files in memory which would prevent moving directories. I happily accept contributions but I am not currently interested in Ansible so I am not going to help actively.
